packages/@glimmer/runtime/test/attributes-test.ts

```diff
@@ -68,204 +62,214 @@ function nativeValueForElementProperty<T extends keyof HTMLElementTagNameMap, P
-});
+    this.rerender({ enabled: 'wat' });
+    this.assertHTML('<input disabled />');
+    this.assertStableNodes();
 
-test("a[href] marks javascript: protocol as unsafe, http as safe", () => {
-  let template = compile('<a href="{{foo}}"></a>');
+    this.rerender({ enabled: null });
+    this.assertHTML('<input />');
+    this.assertStableNodes();
 
-  let context = { foo: 'javascript:foo()' };
-  render(template, context);
+    this.rerender({ enabled: true });
+    this.assertHTML('<input disabled />');
+    this.assertStableNodes();
 
-  equalTokens(root, '<a href="unsafe:javascript:foo()"></a>');
+    this.rerender({ enabled: undefined });
+    this.assertHTML('<input />');
+    this.assertStableNodes();
 
-  rerender({ foo: 'http://foo.bar' });
+    this.rerender({ enabled: true });
+    this.assertHTML('<input disabled />');
+    this.assertStableNodes();
+  }
 
-  equalTokens(root, '<a href="http://foo.bar"></a>');
+  @test "disabled without an explicit value is truthy"(){
+    this.render('<input disabled />');
+    this.assertHTML('<input disabled />');
+    this.assert.ok(readDOMAttr(this.element.firstChild as Element, 'disabled'));
 
-  rerender({ foo: 'javascript:foo()' });
+    this.assertStableRerender();
+    this.assert.ok(readDOMAttr(this.element.firstChild as Element, 'disabled'));
+  }
 
-  equalTokens(root, '<a href="unsafe:javascript:foo()"></a>');
-});
+  @test "a[href] marks javascript: protocol as unsafe"() {
+    this.render('<a href="{{foo}}"></a>', {
+      foo: 'javascript:foo()'
+    });
+    this.assertHTML('<a href="unsafe:javascript:foo()"></a>');
+    this.assertStableRerender();
 
-test("a[href] marks javascript: protocol as unsafe on updates", () => {
-  let template = compile('<a href="{{foo}}"></a>');
+    this.rerender({ foo: 'example.com' });
+    this.assertHTML('<a href="example.com"></a>');
+    this.assertStableNodes();
 
-  let context = { foo: 'http://foo.bar' };
-  render(template, context);
+    this.rerender({ foo: 'javascript:foo()' });
+    this.assertHTML('<a href="unsafe:javascript:foo()"></a>');
+    this.assertStableNodes();
+  }
 
-  equalTokens(root, '<a href="http://foo.bar"></a>');
+  @test "a[href] marks javascript: protocol as unsafe, http as safe"() {
+    this.render('<a href="{{foo}}"></a>', { foo: 'javascript:foo()' });
+    this.assertHTML('<a href="unsafe:javascript:foo()"></a>');
+    this.assertStableRerender();
 
-  rerender({ foo: 'javascript:foo()' });
+    this.rerender({ foo: 'http://foo.bar' });
+    this.assertHTML('<a href="http://foo.bar"></a>');
+    this.assertStableNodes();
 
-  equalTokens(root, '<a href="unsafe:javascript:foo()"></a>');
-
-  rerender({ foo: 'http://foo.bar' });
+    this.rerender({ foo: 'javascript:foo()' });
+    this.assertHTML('<a href="unsafe:javascript:foo()"></a>');
+    this.assertStableNodes();
+  }
 
-  equalTokens(root, '<a href="http://foo.bar"></a>');
-});
+  @test "a[href] marks javascript: protocol as unsafe on updates"() {
```

Comment:
This seems to be covered at [L190](https://github.com/glimmerjs/glimmer-vm/pull/564/files?diff=split#diff-37847c52bf54b71f8378543edd6d9407R190)